src/TicTacToeBoard.java

```diff
@@ -0,0 +1,95 @@
+import java.util.ArrayList;
+
+public class TicTacToeBoard implements IBoard, Finals, IRegistrable {
+
+	private IRubric[] rubrics = new Rubric[BOARD_SIZE];
+	private IPlayer currentPlayer;
+	private CompareRubrics<IRubric> rubricComparator = new CompareRubrics<>();
+	private ArrayList<TicTacToeListener> listeners = new ArrayList<TicTacToeListener>();
+	
+	public TicTacToeBoard(IPlayer currentPlaye) {
```

Comment:
add more player, `currentPlayer` and `otherPlayer`

Reply:
👍